scripts/hypothesistooling.py

```diff
@@ -327,3 +327,81 @@ def update_for_pending_release():
         'commit',
         '-m', 'Bump version to %s and update changelog' % (__version__,)
     )
+
+
+def has_no_effect_on_tests(path):
```

Comment:
This is only ever used negated, so I wonder if it makes sense to invert the logic and have this as `could_affect_tests` instead?